content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-free-function-nodes.md

```diff
@@ -0,0 +1,312 @@
+---
+linktitle: Custom Free Function Nodes
+title: Creating Custom Free Function Nodes in Script Canvas
+description: Learn how to create custom Script Canvas Free Function Nodes in Open 3D Engine (O3DE).
+weight: 200
+---
+
+## Instructions
+This topic guides you how to create custom Script Canvas Free Function Nodes step by step.
+
+{{< note >}}
+The [ScriptCanvasPhysics](https://github.com/o3de/o3de/tree/development/Gems/ScriptCanvasPhysics) Gem demonstrates the finished Gem that creates custom free function nodes in ScriptCanvas.
+You can reference the ScriptCanvasPhysics Gem as you follow along this tutorial.
+{{< /note >}}
+
+### Prerequisite: Adding support for custom free function nodes to a Gem
+In your Gem's `CMakeLists.txt`, add a section for `AUTOGEN_RULES` and declare `Gem::ScriptCanvas.Extensions` as build dependency.
+
+The precise place for this section will vary depending on how your Gem is configured. 
+However, we recommend that your Gem define a `STATIC` library to make the code available to both runtime and editor projects.
+
+As an example, here is partial definition of Gem's `CMakeLists.txt` that supports Script Canvas custom nodes with following required changes:
+1. `Gem::ScriptCanvas.Extensions` must be declared as `BUILD_DEPENDENCIES` of `STATIC` library
+2. Add `AUTOGEN_RULES` section for custom free function under `STATIC` library
+   ```cmake
+   AUTOGEN_RULES
+       *.ScriptCanvasFunction.xml,ScriptCanvasFunctionRegistry_Header.jinja,AutoGenFunctionRegistry.generated.h
+       *.ScriptCanvasFunction.xml,ScriptCanvasFunctionRegistry_Source.jinja,AutoGenFunctionRegistry.generated.cpp
+   ```
+3. `STATIC` library must be declared directly/indirectly as `BUILD_DEPENDENCIES` of Gem runtime/editor module 
```

Comment:
Instead of using slashes ("/") in "directly/indirectly" and "runtime/editor", be explicit about what you mean. Slashes are not always clear to a reader.

Reply:
done